--- input.go
+++ output.go
@@ -1,19 +1,31 @@
+//line input.go:1:1
 package test
 
 import (
-  "database/sql"
+//line input.go:5
   "database/sql/driver"
+//line <generated>:1
+  __orchestrion_sqltrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/database/sql"
 )
 
+//line input.go:8
 var conn driver.Connector
 
 func main() {
-  db1, err := sql.Open("foo", "bar")
+  db1, err :=
+//line <generated>:1
+    __orchestrion_sqltrace.Open(
+//line input.go:11
+      "foo", "bar")
   if err != nil {
     panic(err)
   }
   defer db1.Close()
 
-  db2 := sql.OpenDB(conn)
+  db2 :=
+//line <generated>:1
+    __orchestrion_sqltrace.OpenDB(
+//line input.go:17
+      conn)
   defer db2.Close()
 }
