--- input.go
+++ output.go
@@ -1,3 +1,4 @@
+//line input.go:1:1
 package main
 
 import (
@@ -10,8 +11,22 @@
   "orchestrion/integration"
 )
 
+//line <generated>
+import __orchestrion_chitrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/go-chi/chi.v5"
+
+//line input.go:13
 func main() {
-  router := chi.NewRouter()
+  router :=
+//line <generated>
+    func() *chi.Mux {
+      mux :=
+//line input.go:14
+        chi.NewRouter()
+//line <generated>
+      mux.Use(__orchestrion_chitrace.Middleware())
+      return mux
+    }()
+//line input.go:15
   router.Get("/", func(w http.ResponseWriter, _ *http.Request) {
     w.Write([]byte("Hello World!"))
   })
