--- input.go
+++ output.go
@@ -1,3 +1,4 @@
+//line input.go:1:1
 package main
 
 import (
@@ -9,11 +10,22 @@
   "orchestrion/integration"
 )
 
+//line <generated>
+import __orchestrion_instrument "github.com/datadog/orchestrion/instrument"
+
+//line input.go:12
 func main() {
   s := &http.Server{
-    Addr:    ":8085",
-    Handler: http.HandlerFunc(handle),
+    Addr: ":8085",
+    Handler:
+    //dd:startwrap
+//line <generated>
+    __orchestrion_instrument.WrapHandler(
+//line input.go:15
+      http.HandlerFunc(handle)),
+    //dd:endwrap
   }
+//line input.go:17
   integration.OnSignal(func() {
     s.Shutdown(context.Background())
   })
