--- input.go
+++ output.go
@@ -1,6 +1,27 @@
+//line input.go:1:1
 package test
 
-import "github.com/aws/aws-sdk-go-v2/aws"
+import (
+  "github.com/aws/aws-sdk-go-v2/aws"
 
-var cfgValue = aws.Config{}
-var cfgPtr = &aws.Config{}
+//line <generated>:1
+  __orchestrion_awstrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/aws/aws-sdk-go-v2/aws"
+)
+
+//line input.go:5
+var cfgValue =
+//line <generated>:1
+func(cfg aws.Config) aws.Config {
+  __orchestrion_awstrace.AppendMiddleware(&cfg)
+  return cfg
+}(
+//line input.go:5
+  aws.Config{})
+var cfgPtr =
+//line <generated>:1
+func(cfg *aws.Config) *aws.Config {
+  __orchestrion_awstrace.AppendMiddleware(cfg)
+  return cfg
+}(
+//line input.go:6
+  &aws.Config{})
