--- input.go
+++ output.go
@@ -6,6 +6,8 @@
   "log"
   "net/http"
 
+  __orchestrion_instrument "github.com/datadog/orchestrion/instrument"
+  __orchestrion_event "github.com/datadog/orchestrion/instrument/event"
   "orchestrion/integration"
 )
 
@@ -21,6 +23,26 @@
 }
 
 func handle(w http.ResponseWriter, r *http.Request) {
+  {
+    r = r.WithContext(__orchestrion_instrument.Report(
+      r.Context(),
+      __orchestrion_event.EventStart,
+      "function-name", "handle",
+      "span.kind", "server",
+      "http.method", r.Method,
+      "http.url", r.URL,
+      "http.useragent", r.Header.Get("User-Agent"),
+    ))
+    defer __orchestrion_instrument.Report(
+      r.Context(),
+      __orchestrion_event.EventEnd,
+      "function-name", "handle",
+      "span.kind", "server",
+      "http.method", r.Method,
+      "http.url", r.URL,
+      "http.useragent", r.Header.Get("User-Agent"),
+    )
+  }
   data, err := io.ReadAll(r.Body)
   if err != nil {
     w.WriteHeader(http.StatusBadRequest)
