--- input.go
+++ output.go
@@ -1,13 +1,30 @@
+//line input.go:1:1
 package test
 
 import (
   "context"
+//line <generated>
+  __orchestrion_instrument "github.com/datadog/orchestrion/instrument"
+  __orchestrion_event "github.com/datadog/orchestrion/instrument/event"
 )
 
 //dd:span foo:bar baz:qux
+//line input.go:8
 func outer(ctx context.Context) {
+//line <generated>
+  {
+    __orchestrion_instrument.Report(ctx, __orchestrion_event.EventStart, "name", "outer", "foo", "bar", "baz", "qux")
+    defer __orchestrion_instrument.Report(ctx, __orchestrion_event.EventEnd, "name", "outer", "foo", "bar", "baz", "qux")
+  }
   //dd:span
+//line input.go:10
   inner := func(c context.Context) {
+//line <generated>
+    {
+      __orchestrion_instrument.Report(c, __orchestrion_event.EventStart)
+      defer __orchestrion_instrument.Report(c, __orchestrion_event.EventEnd)
+    }
+//line input.go:11
     panic("Not implemented")
   }
   inner(ctx)
