--- input.go
+++ output.go
@@ -1,3 +1,4 @@
+//line input.go:1:1
 package test
 
 import (
@@ -6,8 +7,12 @@
   "github.com/gomodule/redigo/redis"
 )
 
+//line <generated>
+import __orchestrion_redigo "gopkg.in/DataDog/dd-trace-go.v1/contrib/gomodule/redigo"
+
+//line input.go:9
 func dial(net, address string) error {
-  c, err := redis.Dial(net, address)
+  c, err := __orchestrion_redigo.Dial(net, address)
   if err != nil {
     return err
   }
@@ -16,7 +20,7 @@
 }
 
 func dialContext(ctx context.Context, net, address string) error {
-  c, err := redis.DialContext(ctx, net, address)
+  c, err := __orchestrion_redigo.DialContext(ctx, net, address)
   if err != nil {
     return err
   }
@@ -25,7 +29,7 @@
 }
 
 func dialURL(url string) error {
-  c, err := redis.DialURL(url)
+  c, err := __orchestrion_redigo.DialURL(url)
   if err != nil {
     return err
   }
