--- input.go
+++ output.go
@@ -9,10 +9,33 @@
   "orchestrion/integration"
 )
 
+import (
+  __orchestrion_instrument "github.com/datadog/orchestrion/instrument"
+  __orchestrion_event "github.com/datadog/orchestrion/instrument/event"
+)
+
 func main() {
   s := &http.Server{
     Addr: ":8085",
     Handler: http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+      {
+        r = r.WithContext(__orchestrion_instrument.Report(
+          r.Context(),
+          __orchestrion_event.EventStart,
+          "span.kind", "server",
+          "http.method", r.Method,
+          "http.url", r.URL,
+          "http.useragent", r.Header.Get("User-Agent"),
+        ))
+        defer __orchestrion_instrument.Report(
+          r.Context(),
+          __orchestrion_event.EventEnd,
+          "span.kind", "server",
+          "http.method", r.Method,
+          "http.url", r.URL,
+          "http.useragent", r.Header.Get("User-Agent"),
+        )
+      }
       data, err := io.ReadAll(r.Body)
       if err != nil {
         w.WriteHeader(http.StatusBadRequest)
