--- input.go
+++ output.go
@@ -5,6 +5,8 @@
   "log"
   "net/http"
 
+  __orchestrion_instrument "github.com/datadog/orchestrion/instrument"
+  __orchestrion_event "github.com/datadog/orchestrion/instrument/event"
   "orchestrion/integration"
 )
 
@@ -19,5 +21,25 @@
   log.Printf("Server shut down: %v", s.ListenAndServe())
 }
 
-func handle(http.ResponseWriter, *http.Request) {
+func handle(_ http.ResponseWriter, __argument__1 *http.Request) {
+  {
+    __argument__1 = __argument__1.WithContext(__orchestrion_instrument.Report(
+      __argument__1.Context(),
+      __orchestrion_event.EventStart,
+      "function-name", "handle",
+      "span.kind", "server",
+      "http.method", __argument__1.Method,
+      "http.url", __argument__1.URL,
+      "http.useragent", __argument__1.Header.Get("User-Agent"),
+    ))
+    defer __orchestrion_instrument.Report(
+      __argument__1.Context(),
+      __orchestrion_event.EventEnd,
+      "function-name", "handle",
+      "span.kind", "server",
+      "http.method", __argument__1.Method,
+      "http.url", __argument__1.URL,
+      "http.useragent", __argument__1.Header.Get("User-Agent"),
+    )
+  }
 }
